server/src/models.js

```diff
@@ -64,6 +76,14 @@ UserModel.belongsTo(OrganisationModel);
 // devices belong to a single user
 DeviceModel.belongsTo(UserModel);
 
+//users -> tags
+UserModel.belongsToMany(TagsModel, { through: 'tag_user' });
```

Comment:
I think you want to swap these around:
```
TagsModel.belongsToMany(UserModel,{ through: 'user_tag' });
```
ultimately it might work out the same in the DB but I think it makes it easier to reason about. (the _user_ has the tag, not the _tag_ has the user)